packages/runner/src/Runner.ts

```diff
@@ -53,9 +61,10 @@ export class Runner
 
     /**
      * Dispatch/Broadcast Runner to all listeners added to the queue.
-     * @param {...any} params - optional parameters to pass to each listener
+     * @param {...any} [params] - optional parameters to pass to each listener
```

Comment:
For me making the params optional gives an error in the types file.

`A rest parameter cannot be optional.`